### PR TITLE
[AND-193] Add Powered by AI

### PR DESCRIPTION
**Pull request by tomascarvalho7**
**What does this PR do?**

Add powered by AI on top of the chat

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] PoweredByAi.kt

**How should this be manually tested?**

Open GameGenie and check it's there

  Flow on how to test this or QA Tickets related to this use-case: [AND-193](https://aptoide.atlassian.net/browse/AND-193)

**What are the relevant tickets?**

  Tickets related to this pull-request: [AND-193](https://aptoide.atlassian.net/browse/AND-193)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 

**Code Review Checklist**

- [ ] Documentation on public interfaces
- [ ] Database changed? If yes - Migration?
- [ ] Remove comments & unused code & forgotten testing Logs
- [ ] Codestyle
- [ ] New Kotlin code has unit tests
- [ ] New flows in presenters unit tests
- [ ] Mappers/Validators with any kind of logic unit tests
- [ ] Functional tests pass

[AND-193]: https://aptoide.atlassian.net/browse/AND-193?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[AND-193]: https://aptoide.atlassian.net/browse/AND-193?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by apkbuilder-lw** [comment]
Can one of the admins verify this patch?

**Comment by apkbuilder-lw** [comment]
Can one of the admins verify this patch?

File: app-games/src/main/java/com/aptoide/android/aptoidegames/gamegenie/presentation/composables/PoweredByAi.kt [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+@Composable
+fun PoweredByAi() {
+  return Row(
+    modifier = Modifier
+      .fillMaxWidth()
+      .padding(bottom = 32.dp),
+    horizontalArrangement = Arrangement.Center
+  ) {
+    Text(
+      text = "Powered by AI",
```
Comment (phillipsgoncalves) [inline app-games/src/main/java/com/aptoide/android/aptoidegames/gamegenie/presentation/composables/PoweredByAi.kt:23]: hardcoded text, we are waiting for translations.

**Review by phillipsgoncalves — CHANGES_REQUESTED** [review]

**Comment by phillipsgoncalves** [comment]
Since the other 1st commit is here, i cant run because the commit is broken

**Comment by phillipsgoncalves** [comment]
This pull request will be closed, since we are reviewing everything at the same time on :
https://github.com/Aptoide/aptoide-client-v8/pull/2156

This solution was adopted because all branches came out of each other so this way we can check everything in a row without having to solve conflicts or rebase 3 times.

**Pull request closed without merging**